doc/rst/language/spec/ranges.rst

```diff
@@ -168,16 +171,14 @@ The type of a range is characterized by three parameters:
 
 -  ``stridable`` is a boolean that determines whether the range’s stride
    can take on values other than 1. ``stridable`` is ``false`` by
-   default. A range is called *stridable* if its type’s ``stridable`` is
-   ``true``.
+   default. A range is called *stridable* if its type’s ``stridable``
+   value is ``true``.
```

Comment:
(Are you playing evil games to set the ground for replacing `param` with `value`? :D)

Consider "field" instead of "value".

Reply:
No, just thought the lack of a noun was problematic and thought that 'value' seemed more precise than field.  But I'm fine changing it.